guides/v2.1/config-guide/multi-site/ms_nginx.md

```diff
@@ -51,11 +52,54 @@ Setting up multiple stores consists of the following tasks:
     *   `$MAGE_RUN_CODE` is the unique website or store view code that corresponds to `$MAGE_RUN_TYPE`.
 
 ## Step 2: Create nginx virtual hosts {#ms-nginx-vhosts}
-This section discusses how to load websites on the {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %}. You can use either websites or store views; if you use store views, you must adjust parameter values accordingly. You must complete the tasks in this section as a user with `root` privileges.
+This section discusses how to load websites on the {% glossarytooltip 1a70d3ac-6bd9-475a-8937-5f80ca785c14 %}storefront{% endglossarytooltip %}. You can use either websites or store views; if you use store views, you must adjust parameter values accordingly. You must complete the tasks in this section as a user with at least `sudo` privileges.
 
-{% collapsible To create virtual hosts: %}
+You can use one [nginx virtual host](#ms-nginx-vhosts) or many [nginx virtual host](#ms-nginx-vhosts) to load multiple websites. Using just one virtual host file allows you too keep your [nginx virtual host](#ms-nginx-vhosts) file simple and clean, and using many [nginx virtual host](#ms-nginx-vhosts) allows customizations for each particular store, such as a custom location for `french.mysite.mg`.
```

Comment:
Consider a more imperative mood for the last two sentences, something like: "By using just one virtual host file, you can keep your [nginx virtual host](#ms-nginx-vhosts) file simple and clean. By using many [nginx virtual hosts](#ms-nginx-vhosts), you can customize each store, such as a custom location for `french.mysite.mg`

Reply:
👍